### PR TITLE
mammals object coding in js

### PR DESCRIPTION
**Pull request by Moon2024-light**
hi,team please review my object coding in mammals js file please inform me if there is correction needed.thanks

### CODE REVIEW

File: scripts/mammals.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const mammals = [
+    {
+      name: "Tasmanian Devil",
+      group: "mammals",
+     image: "'\images\transparent-img\tasmanian-devil.png'",
+      shortDescription: "A carnivorous marsupial found in Tasmania, known for its loud screeches...",
+      fullDescription: "The Tasmanian Devil is the world's largest surviving carnivorous marsupial. Native to Tasmania, these nocturnal animals are known for their powerful jaws and fierce temperament. Despite their reputation, they play a crucial role in maintaining ecosystem balance.",
```
Comment (jasminedns) [inline scripts/mammals.js]: indent the lines

**Review by jasminedns — CHANGES_REQUESTED** [review]

File: scripts/mammals.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+const mammals = [
+    {
+      name: "Tasmanian Devil",
+      group: "mammals",
+     image: "'\images\transparent-img\tasmanian-devil.png'",
```
Comment (TobiasKjernell) [inline scripts/mammals.js]: this url link is going to be inserted in img-htm sourcel. so have to think from the html perspective path. ex: ..\images\pic...

**Review by jasminedns — APPROVED** [review]

**Review by TobiasKjernell — CHANGES_REQUESTED** [review]

File: scripts/mammals.js (hunk 2 above)
Comment (Moon2024-light) [reply scripts/mammals.js]: hi,i made the changes please have a look thanks

**Review by TobiasKjernell — APPROVED** [review]

**Comment by TobiasKjernell** [comment]
looks good

**Pull request merged**